tools/depends/target/binary-addons/Makefile

```diff
@@ -1,5 +1,5 @@
 BUILDDIR := $(shell pwd)
-ADDONS := all
+ADDONS ?= all
```

Comment:
Just noticed this also takes the empty value if it is set and `make ADDONS=<ID>` doesn't work anymore.
@wsnipex you have a better suggestion?

Reply:
this should work.
Alternatively you'd have to use a temporary var here, that is not overwriteable by the user/env

Reply:
As it did the same (taking the empty value) already before this change, all fine.